src/test/SkipCallsTestHook.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.24;
+
+import {Hooks} from "../libraries/Hooks.sol";
+import {BaseTestHooks} from "./BaseTestHooks.sol";
+import {IHooks} from "../interfaces/IHooks.sol";
+import {IPoolManager} from "../interfaces/IPoolManager.sol";
+import {PoolKey} from "../types/PoolKey.sol";
+import {BalanceDelta} from "../types/BalanceDelta.sol";
+import {PoolId, PoolIdLibrary} from "../types/PoolId.sol";
+import {FeeLibrary} from "../../src/libraries/FeeLibrary.sol";
+import {IERC20Minimal} from "../interfaces/external/IERC20Minimal.sol";
+import {CurrencyLibrary, Currency} from "../types/Currency.sol";
+import {PoolTestBase} from "./PoolTestBase.sol";
+import {Test} from "forge-std/Test.sol";
+
+contract SkipCallsTestHook is BaseTestHooks, Test {
+    using PoolIdLibrary for PoolKey;
+    using Hooks for IHooks;
+    using FeeLibrary for uint24;
+
+    uint256 public counter;
+    IPoolManager manager;
+    uint24 internal fee;
+
+    function setManager(IPoolManager _manager) external {
+        manager = _manager;
+    }
+
+    function setFee(uint24 _fee) external {
```

Comment:
These are unneeded functions i believe for the purpose of this test

Reply:
Oh ya i needed them since I made the hook with a dynamic fee but ill change that so i dont need these